.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - 1.7
+
+env:
+  - CGO_ENABLED=0
+
+script:
+  - go vet
+  - go test
+  - go build -ldflags "-s -w -X main.build=$TRAVIS_BUILD_NUMBER -X main.rev=$TRAVIS_COMMIT" -a -tags netgo
```

Comment:
Ah, I think this drifted. Updated in 868059a89c9216d0e35854399b39c13c34c3096c. Thanks.